packages/components/psammead-episode-list/src/Episode.jsx

```diff
@@ -34,7 +34,7 @@ const Episode = ({ children, dir }) => {
   return (
     <Wrapper dir={dir} showMediaIndicator={showMediaIndicator}>
       {showMediaIndicator ? (
-        Children.map(children, child =>
+        Children.filter(Boolean).map(children, child =>
```

Comment:
Neat. Didn't know you could do this 👌

Reply:
CI fail on this... looks like this can't be called if Children isn't an array? i.e. undefined

`TypeError: _react.Children.filter is not a function`